### PR TITLE
Falco fork with 6.3 fixes

### PR DESCRIPTION
**Pull request by erthalion**
## Description

Apply 6.3 fixes to the current version of Falco fork.

## Checklist
- [ ] Investigated and inspected CI test results

If any of these don't apply, please comment below.

## Testing Performed

CI is enough

### CODE REVIEW

**Comment by Molter73** [comment]
What's the corresponding branch on our fork? Do we have some tag or other way of searching for it in a couple months if we need to fix something.

Also, I accidentally built the `2.6.0-rc1` drivers when testing #1248 , could you use `2.6.0-rc2` to ensure the drivers are properly built?

Also also, we probably want to run the PR with the `all-integration-tests` label to ensure Fedora is properly fixed (I'm pretty sure it is, but still).

**Comment by erthalion** [comment]
> What's the corresponding branch on our fork of Falco? Do we have some tag or other way of searching for it in a couple months if we need to fix something?

It's been a while, but now things slowly coming back. My intentions were to test the fixes first on this PR, then merge the fixes in the Falco fork: https://github.com/stackrox/falcosecurity-libs/pull/54 . Could I get a +1 there?

**Comment by Molter73** [comment]
> > What's the corresponding branch on our fork of Falco? Do we have some tag or other way of searching for it in a couple months if we need to fix something?
> 
> It's been a while, but now things are slowly coming back. My intentions were to test the fixes first on this PR, then merge the fixes in the Falco fork: [stackrox/falcosecurity-libs#54](https://github.com/stackrox/falcosecurity-libs/pull/54) . Could I get a +1 there?

Approved!

**Comment by Molter73** [comment]
There are 2 debian kernels failing to build due to the known issue where kernel-packer deletes the bundles, feel free to add them to the BLOCKLIST file.

File: kernel-modules/BLOCKLIST [hunk 1]
```diff
@@ -65,7 +65,6 @@
 6.2.* 2.1.0 *
 6.2.* 2.0.1 *
 # TODO(ROX-16705) - 6.3/6.4 kernel compilation errors
-6.3.*
 6.4.*
```
Comment (Molter73) [inline kernel-modules/BLOCKLIST:68]: You can also remove `6.4.*` from this list, I tested a 6.4 kernel on Fedora locally and the CO-RE BPF probe worked like a charm

**Review by Molter73 — APPROVED** [review]
LGTM!

**Pull request merged**